content/news/events-february-2025.md

```diff
@@ -0,0 +1,61 @@
++++
+headline = "ARTICLE"
+title = "Foulab Events - February 2025"
+lastmod = "2025-01-30 21:52:33-0400"
+++++
+
+{{< centered-image src="/events-february-2025/champagneGatorade.png" >}}
```

Comment:
Image references should be with a simple filename, no path. Example: https://github.com/FOULAB/foulab-dot-org-hugo/blob/91dfe26930ac1f56b10b645a4979d480f941f647/content/news/events-january-2025.md?plain=1#L7